.clomonitor.yaml

```diff
@@ -0,0 +1,7 @@
+# CLOMonitor metadata file
+# This file must be located at the root of the repository
+
+# Checks exemptions
+exemptions:
+  - check: recent_release # Check identifier (see https://github.com/cncf/clomonitor/blob/main/docs/checks.md#exemptions)
+    reason: "There are no releases planned for GitHub. Releases for images are done on a rolling basis to quay.io."
```

Comment:
We could potentially be explicit here and state that `Releases for images are done on a rolling basis to quay.io at quay.io/devfile/base-developer-image:ubi8-latest and quay.io/devfile/universal-developer-image:ubi8-latest.`

Though this information is already present in the README and if we ever change the image tag we'd have to remember to update it here.

@svor @ibuziuk @l0rd WDYT?

Reply:
@svor +1 that sounds good to me.

@Jdubrick When you have a moment, can you please change the `reason`field to @svor's suggestion please? Then we can get this merged. Thank you!

Reply:
@AObuchow all done!